templates/workspace_services/sql/terraform/mysql.tf

```diff
@@ -0,0 +1,62 @@
+resource "random_password" "password" {
+  length      = 20
+  min_upper   = 2
+  min_lower   = 2
+  min_numeric = 2
+  min_special = 2
+}
+
+resource "azurerm_mysql_server" "mysql" {
+  name                              = "mysql-${local.service_resource_name_suffix}"
+  resource_group_name               = data.azurerm_resource_group.ws.name
+  location                          = data.azurerm_resource_group.ws.location
+  administrator_login               = "mysqladmin"
+  administrator_login_password      = random_password.password.result
+  sku_name                          = local.sql_sku[var.sql_sku].value
+  storage_mb                        = var.storage_mb
+  version                           = "8.0"
+  auto_grow_enabled                 = true
+  backup_retention_days             = 7
+  geo_redundant_backup_enabled      = false
+  infrastructure_encryption_enabled = false
+  public_network_access_enabled     = false
+  ssl_enforcement_enabled           = true
+  ssl_minimal_tls_version_enforced  = "TLS1_2"
+
+  lifecycle { ignore_changes = [tags] }
+}
+
+resource "azurerm_mysql_database" "db" {
+  name                = var.db_name
+  resource_group_name = data.azurerm_resource_group.ws.name
+  server_name         = azurerm_mysql_server.mysql.name
+  charset             = "utf8"
+  collation           = "utf8_unicode_ci"
+}
```

Comment:
A question: do we have to create it? can't a user connect to the server and create their own db (via mysql cli)?

Reply:
I believe it's possible to create one with command line just thought it would be more user friendly like that. @marrobi 
WDYT?

Reply:
I think you need to use the `az` cli to create a database. Rather than the `mysql` cli. this is an Azure specific thing, so weshould pre create a DB as per the PR.

https://docs.microsoft.com/en-us/azure/mysql/single-server/quickstart-create-mysql-server-database-using-azure-cli

Reply:
I remember creating databases via the mysql cli (when dealing with the Gitea service).
Anyway, we can create it as in the PR...

Reply:
So as decided, I am leaving it as it is...